coil-sample/src/main/assets/webps.json

```diff
@@ -0,0 +1,2966 @@
+[
```

Comment:
Can you remove these? I don't think we need a separate section for webps. If you're up for it it would be better to add a regression test in `BitmapFactoryDecoderTest` (though no worries if you'd rather not).

Reply:
Not sure how the test would work. Most of the time the app does not crash but images are just partially decoded. Bitmaps have the correct size so the test would have to look at the pixels directly.

Reply:
Coil's tests have a `Bitmap.isSimilarTo` extension function to test the visual similarity of images. You could use that to compare a jpg with a webp of the same image.

Reply:
I wrote a simple test for lossy WebP decoding. I had a problem because `Bitmap.isSimilarTo` returned true even when the WebP decoding failed on API 17 and the bitmaps were definitely not similar. It turns out there was a bug in the `crossCorrelation()` function so I changed it too.